src/js/dp/search.js

```diff
@@ -304,4 +304,34 @@ if (searchContainer) {
       });
     });
   };
-}
+
+  //capture focus in checkboxes
+  const showResultsBtn = document.getElementById('show-results');
+  const checkboxes = document.querySelectorAll('input[type="checkbox"]:not([disabled])');
+  const firstCheckbox = checkboxes[0];
+  const keycodeTab = 9;
+
+  if (showResultsBtn) {
+    document.addEventListener('keydown', function (e) {
+      if (e.key === 'Tab' || e.code === keycodeTab.toString()) {
+        if (document.activeElement === showResultsBtn) {
+          firstCheckbox.focus();
```

Comment:
This is blocking access to things above checkboxes - for example if the topic filters are switched off, you can't access the Content Type collapsible or the clear all